lib/bolt_spec/plans/action_stubs.rb

```diff
@@ -55,7 +57,10 @@ def assert_called(object)
           end
           message = "Expected #{object} to be called #{times} times"
           message += " with targets #{@invocation[:targets]}" if @invocation[:targets]
-          message += " with parameters #{parameters}" if parameters
+          if parameters
+            parameters_str = JSON.parse(parameters.to_json)
```

Comment:
This seems fine but probably should have a comment. We probably should discuss whether to add better `.inspects` to the plan language datatype classes.

Reply:
Added a comment here.